include/ruby/internal/has/builtin.h

```diff
@@ -89,16 +96,20 @@
 # define RBIMPL_HAS_BUILTIN___builtin_clzll             HAVE_BUILTIN___BUILTIN_CLZLL
 # define RBIMPL_HAS_BUILTIN___builtin_constant_p        HAVE_BUILTIN___BUILTIN_CONSTANT_P
 # define RBIMPL_HAS_BUILTIN___builtin_ctz               HAVE_BUILTIN___BUILTIN_CTZ
+# define RBIMPL_HAS_BUILTIN___builtin_ctzl              0
 # define RBIMPL_HAS_BUILTIN___builtin_ctzll             HAVE_BUILTIN___BUILTIN_CTZLL
 # define RBIMPL_HAS_BUILTIN___builtin_expect            HAVE_BUILTIN___BUILTIN_EXPECT
 # define RBIMPL_HAS_BUILTIN___builtin_mul_overflow      HAVE_BUILTIN___BUILTIN_MUL_OVERFLOW
 # define RBIMPL_HAS_BUILTIN___builtin_mul_overflow_p    HAVE_BUILTIN___BUILTIN_MUL_OVERFLOW_P
 # define RBIMPL_HAS_BUILTIN___builtin_popcount          HAVE_BUILTIN___BUILTIN_POPCOUNT
+# define RBIMPL_HAS_BUILTIN___builtin_popcountl         0
+# define RBIMPL_HAS_BUILTIN___builtin_rotateleft32      0
+# define RBIMPL_HAS_BUILTIN___builtin_rotateleft64      0
+# define RBIMPL_HAS_BUILTIN___builtin_rotateright32     0
+# define RBIMPL_HAS_BUILTIN___builtin_rotateright64     0
 # define RBIMPL_HAS_BUILTIN___builtin_popcountll        HAVE_BUILTIN___BUILTIN_POPCOUNTLL
 # define RBIMPL_HAS_BUILTIN___builtin_sub_overflow      HAVE_BUILTIN___BUILTIN_SUB_OVERFLOW
-# if defined(UNREACHABLE)
-#  define RBIMPL_HAS_BUILTIN___builtin_unreachable 1
-# endif
+# define RBIMPL_HAS_BUILTIN___builtin_unreachable       defined(UNREACHABLE)
```

Comment:
Seems this should also be worked around.

Reply:
Because of potential -Wexpansion-to-defined, right, I'll fix it.